server/tests-py/test_v1_queries.py

```diff
@@ -535,6 +535,10 @@ def test_export_replace(self, hge_ctx):
         }
         replace_code, replace_resp, _ = hge_ctx.anyq(url, replace_query, headers)
         assert replace_code == 200, replace_resp
+        # The export_resp and export_resp_1 should be the same
```

Comment:
It will greatly help to have an explanation on **why** we are doing this check, the `assert export_resp == export_resp_1` [already conveys this comment](https://github.com/hasura/graphql-engine/blob/master/server/STYLE.md#inline-comments).

Reply:
I have added some comments to explain the test.